Content.Server/_Goobstation/StationEvents/Metric/Components/CombatMetricComponent.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Content.Server.StationEvents.Events;
+using Content.Shared.FixedPoint;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Generic;
+
+namespace Content.Server._Goobstation.StationEvents.Metric.Components;
+
+[RegisterComponent, Access(typeof(CombatMetricSystem))]
+public sealed partial class CombatMetricComponent : Component
+{
+    [DataField, ViewVariables(VVAccess.ReadWrite)]
```

Comment:
`Data field HostileScore in data definition CombatMetricComponent has ViewVariables attribute with ReadWrite access, which is redundant`
Same for the other fields
```suggestion
    [DataField]
```